docs/Usage.md

```diff
@@ -272,6 +273,39 @@ kubectl apply -f cluster-crs.yaml
 kubectl delete cluster proxmox-quickstart
 ```
 
+
+#### Provision a cluster with flatcar
```

Comment:
```suggestion
#### Provision a cluster with Flatcar Container Linux Images
```

Reply:
For people who have never heard of Flatcar to ensure they know it's about a different linux distribution